src/Components/Capture.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Capture.h"
+#include "Render.h"
+#include "Encoder.h"
```

Comment:
Please order your include according to the style guide:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes